payment.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1"> <!-- Ensures optimal rendering on mobile devices -->
+    <title>Make Payment</title>
+  </head>
+
+  <body>
+    <!-- Include the PayPal JavaScript SDK; replace "test" with your own sandbox Business account app client ID -->
+    <script src="https://www.paypal.com/sdk/js?client-id=AR5RGg6eq2jEloMZYTYdXSZUFLqEze-0EpDN54UE3j7uY2PIjXcXgpyh3gHJaOXJwLKdYinCRlC8IUsd&currency=USD"></script>
+
+    <!-- Set up a container element for the button -->
+    <div id="paypal-button-container"></div>
+
+    <script>
```

Comment:
do we want the script section to be in its own file and then we load it in?

Reply:
In this case it needs to be nested like this due to line 10 that holds the Paypal SDK

Reply:
ah okay, thank you for the clarification